util/imageutil/config.go

```diff
@@ -19,7 +19,10 @@ type IngesterProvider interface {
 	content.Provider
 }
 
-func Config(ctx context.Context, str string, resolver remotes.Resolver, ingester IngesterProvider) (digest.Digest, []byte, error) {
+func Config(ctx context.Context, str string, resolver remotes.Resolver, ingester IngesterProvider, platform string) (digest.Digest, []byte, error) {
```

Comment:
@ijc @tonistiigi I wonder if it's time to make a struct as a parameter, or options or something else than specifying an empty string for platform. I find it low in information, WDYT?